libraries/resource_helpers.rb

```diff
@@ -17,6 +17,10 @@ def splunk_bin_path
       Pathname.new(install_dir).join('bin')
     end
 
+    def splunk_app_path
```

Comment:
If we're only calling this within the single app resource, why are we breaking it out into a helper? 

(also this is for locally installed apps as opposed to SHC Deployer, Indexer Cluster Master, or Deployment Server pushed apps).